addons/product/models/product_pricelist_item.py

```diff
@@ -186,6 +220,14 @@ def _compute_rule_tip(self):
                     item.env, discounted_price + item.price_surcharge, item.currency_id),
             )
 
+    def _get_integer(self, percentage):
+        return int(percentage) if percentage.is_integer() else percentage
+
+    def _get_displayed_discount(self, item):
+        if item.base == 'standard_price':
+            return "markup", self._get_integer(item.price_markup)
+        return "discount", self._get_integer(item.price_discount)
+
```

Comment:
not translated ? :thinking:

Reply:
There should be no pot update in master, it'll be generated after the freeze for all the modules.
I meant that you're not using the translation method here.

Reply:
Yes, or it won't be translated ;)